README.adoc

```diff
@@ -62,7 +62,7 @@ Minishift is an open-source project dedicated to developing and supporting Minis
 The code base is forked from the https://github.com/kubernetes/minikube[Minikube] project.
 
 Contributions, questions, and comments are all welcomed and encouraged!
-Minishift developers hang out on IRC in the #minishift channel on Freenode.
+Join the Minishift team on the #minishift IRC channel on  https://freenode.net/[Freenode] or sign up for the https://lists.minishift.io/admin/lists/minishift.lists.minishift.io[Minishift Mailing List].
```

Comment:
> Also, this is not an "or" situation. It is better to say, maybe in an amicable way, "You can also signup"

not sure, I don't see this or in the sense of "either ... or". I would not read in a way where I need to make a choice for one of them.